be/src/storage/lake/rowset.cpp

```diff
@@ -67,6 +90,33 @@ Rowset::~Rowset() {
     }
 }
 
+void Rowset::add_uncompacted_segments(TxnLogPB_OpCompaction* op_compaction, uint64_t& uncompacted_num_rows,
+                                      uint64_t& uncompacted_data_size) const {
+    if (!partial_segments_compaction()) { // defensive check
```

Comment:
this is an internal state consistency check, may just turn it to DCHECK()?

Reply:
done